drawee/src/main/java/com/facebook/drawee/view/DraweeView.java

```diff
@@ -234,6 +234,15 @@ protected void onMeasure(int widthMeasureSpec, int heightMeasureSpec) {
   }
 
   @Override
+  protected void onSizeChanged(int w, int h, int oldw, int oldh) {
+    super.onSizeChanged(w, h, oldw, oldh);
+    Drawable drawable = getTopLevelDrawable();
+    if (drawable != null) {
+      drawable.setBounds(0, 0, w, h);
```

Comment:
Unfortunately this doesn't account for potential padding set on the view. We need to account for that too.